requirements/requirements.txt

```diff
@@ -87,7 +86,7 @@ jsonobject-couchdbkit==0.9.16
 jsonobject==0.9.8
 jsonpath-rw==1.4.0
 kafka-python==1.4.4
-kombu==3.0.37             # via celery
+kombu==4.3.0              # via celery
```

Comment:
Probably doesn't affect us since we use rabbitmq but this bug is a thing: https://github.com/celery/kombu/issues/1006

Reply:
This does prevent you from running celery locally, so I'm going to pin the version until the fix is released: 849ac40